### DISCUSSION TITLE
Very curious where I can download or follow Bitwarden Desktop v2024.12.0

### DISCUSSION THREAD

**Discussion by somnifex** [dani-garcia/vaultwarden#5207] [Q&A] [answered]
I noticed that the official website only released 11.0, but the latest update log of vaultwarden suggests that 12.0 supports SSH storage.

**Comment by PrieserMax** [comment]
+1

**Comment by BlackDex** [comment, marked as answer]
It will be available for download once it's ready. It isn't even December yet. You can build it your self or try a GitHub action build version from the Bitwarden Client repo. But that is all on your own risk of course.
